q2_emperor/_plot.py

```diff
@@ -60,6 +61,12 @@ def generic_plot(output_dir: str, master: skbio.OrdinationResults,
 def plot(output_dir: str, pcoa: skbio.OrdinationResults,
          metadata: qiime2.Metadata, custom_axes: str = None,
          ignore_missing_samples: bool = False) -> None:
+
+    if pcoa.features is not None:
+        pcoa.features = None
+        warnings.warn('Arrows cannot be visualized with the plot method, use '
+                      'biplot instead')
```

Comment:
Sorry for the slow review, @ElDeveloper! Warnings won't necessarily be shown to users that are running q2cli without enabling `--verbose` mode - do you think that matters here? 

If it does matter, then perhaps an alternative solution is to expose a new `ignore_pcoa_features` (or something like that):

```python
def plot(output_dir: str, pcoa: skbio.OrdinationResults,
         metadata: qiime2.Metadata, custom_axes: str = None,
         ignore_missing_samples: bool = False,
         ignore_pcoa_features: bool = False) -> None:

if ignore_pcoa_features:
    pcoa.features = None

if pcoa.features is not None:
   raise ValueError("what on earth are you doing?!")

...
```

Reply:
Great suggestion!